ui/lib/block.jsx

```diff
@@ -168,129 +162,142 @@ const Header = {
                 }
             }
         }
-        let title = m("div[contentEditable]", style("inner", handlers), m.trust(vnode.attrs.title));
-        if (vnode.attrs.flow === true) {
-            return m("div", style(["flow", "outer"]), [
-                (vnode.attrs.inflow) ? m(InflowEndpoint, { id: vnode.attrs.id + "-in" }) : undefined,
+        let title = <div contentEditable
+            class="title"
+            style={inner}
+            oninput={handlers.oninput}
+            ondblclick={handlers.ondblclick}>
+            {m.trust(attrs.title)}
```

Comment:
I am not sure if it's intended or not, but not escaping user inputs could lead to a possible XSS security issue.

https://mithril.js.org/trust.html